### PR TITLE
docs: enhance docs for merging modifiers

### PR DESCRIPTION
**Pull request by cynful**
#### Description
- Enhances docs for merging modifiers

#### This PR fixes the following issues
n/a

### CODE REVIEW

File: docs/workflow.md [hunk 1]
```diff
@@ -353,8 +353,11 @@ data: # final
   foo_param: "a string"
 ```
 
-We can change the behavior by using merging modifiers in the names of overriding data
+We can change the behavior by using merging modifiers at the end of the overriding data names.
 
+Usage:
+
+`var` is an example name of the overriding data, the following character is indicates what type of merge modifier to use.
```
Comment (wyardley) [inline docs/workflow.md]: "is indicates"?

Comment (cynful) [reply docs/workflow.md]: 🤦‍♀

**Pull request merged**